uk.co.compucorp.civicrm.hrleaveandabsences/js/test/leave-type-wizard/components/leave-type-wizard.component.spec.js

```diff
@@ -207,6 +207,28 @@ define([
             expect(controller.sections[0].active).toBe(true);
           });
         });
+
+        describe('when user clicks the "next" button 2 times and then "back" 1 time', function () {
+          beforeEach(function () {
+            controller.goNext();
+            controller.goNext();
+            controller.goBack();
+          });
+
+          it('opens seconds tab', function () {
+            expect(secondSection.tabs[1].active).toBe(true);
+          });
+
+          describe('when user clicks the "next" button', function () {
+            beforeEach(function () {
+              controller.goNext();
+            });
+
+            it('opens seconds tab', function () {
```

Comment:
The `it` seems a bit confusing. It opens second tab with `tabs` index of `2` while the previous `it` scope says the same thing with `tabs` index of `1`.

Reply:
Fixed.